lib/src/main/java/edu/stanford/futuredata/macrobase/analysis/summary/fpg/FPGExplanation.java

```diff
@@ -0,0 +1,112 @@
+package edu.stanford.futuredata.macrobase.analysis.summary.fpg;
+
+import edu.stanford.futuredata.macrobase.analysis.summary.Explanation;
+import edu.stanford.futuredata.macrobase.analysis.summary.fpg.result.FPGAttributeSet;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents a summarization result, which contains a list of attribute values
+ * and other statistics about the underlying process, e.g. num of tuples observed
+ * so far.
+ */
+public class FPGExplanation implements Explanation {
```

Comment:
Kept around for backwards compatibility until we move fpgrowth to use the new explanations in a future PR